cni-plugins/sriov-passthrough-cni/README.md

```diff
@@ -27,3 +27,37 @@ plugin on all the relevant K8s nodes.
 │   └── sriov-passthrough-cni
 └── README.md
 ```
+
+## Prow job usage:
+
+### In order to use the CNI and allocate one sriov PF:
+1. Add the following annotation to the prow job yaml (or leave the current annotation unchanged).
+
+`k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}`
+
+2. When the parallel sriov jobs mode is enabled, add the following requests/limits:
+```
+requests:
+  prow/sriov: "1"
+limits:
+  prow/sriov: "1"
+```
+
+### In order to run the CNI twice, so 2 sriov PFs would be allocated per job:
```

Comment:
I think this headline, and the previous one should be about *what* will be detailed further on - allocate 2 (or 1) PF. The *how* should be in the description beneath  the headline.
something like: 

```
### Allocate 2 SR-IOV PFs per job:

One will have to run the cni twice in order for 2 SR-IOV PFs to be allocatable per job:
+1. USe the following ....
```

Strating a headline with "in order" seems weird. what do you think?